### PR TITLE
Explicit error message when setting active a Viewport that is already active

### LINKED ISSUE #50382: Non explicit error message when setting active a `Viewport` that is already active
### Godot version

4.0 dev, 3.x

### System information

All

### Issue description

When calling `VisualServer::viewport_set_active(viewport, true)` on a `Viewport` that is already active, the engine prints the following non explicit error message (tested with Godot 3.1.2, but true on all 3.x and 4.0-dev) :
```
E 0:00:05:0437   Condition ' active_viewports.find(viewport) != -1 ' is true.
  <C Source>     servers/visual/visual_server_viewport.cpp:369 @ viewport_set_active()
```

This message is triggered as following ([see actual code](https://github.com/godotengine/godot/blob/a2d5f191d8d91f241a09233586da0f7e9deb8a76/servers/rendering/renderer_viewport.cpp#L697)) :
`ERR_FAIL_COND(active_viewports.find(viewport) != -1); //already active`

The message should be more explicit, for instance `Can't make active a Viewport that is already active`.
By the way I'm wondering if any error is supposed to be raised at all here (should be preferable not to fail and do nothing silently ?)

### Steps to reproduce

Make active any `Viewport` that is already active.

### Minimal reproduction project

_No response_

### PR DESCRIPTION
**Pull request by viteksykora**
Fixes #50382

### CODE REVIEW

File: servers/rendering/renderer_viewport.cpp [hunk 1]
```diff
@@ -694,7 +694,7 @@ void RendererViewport::viewport_set_active(RID p_viewport, bool p_active) {
 	ERR_FAIL_COND(!viewport);
 
 	if (p_active) {
-		ERR_FAIL_COND(active_viewports.find(viewport) != -1); //already active
+		ERR_FAIL_COND_MSG(active_viewports.find(viewport) != -1 , "Can't make active a Viewport that is already active" ); //already active
```
Comment (Calinou) [inline servers/rendering/renderer_viewport.cpp:697]:
```suggestion
		ERR_FAIL_COND_MSG(active_viewports.find(viewport) != -1, "Can't make active a Viewport that is already active.");
```

**Review by Calinou — DISMISSED** [review]

**Comment by akien-mga** [comment]
Looks good, please squash the two commits together (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase)).

**Comment by TaskManagerCZ** [comment]
> Looks good, please squash the two commits together (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase)).

done :)

**Comment by akien-mga** [comment]
Sorry I should have commented on this earlier, could you also amend the commit message to be more explicit about what it's changing? Mentioning line number is not useful btw, it's included in the diff itself and this will be outdated information as soon as a new line is added or removed in the file.

Something like this PR's title would be a clearer commit message: "Explicit error message when setting active a Viewport that is already active"

**Comment by TaskManagerCZ** [comment]
> Sorry I should have commented on this earlier, could you also amend the commit message to be more explicit about what it's changing? Mentioning line number is not useful btw, it's included in the diff itself and this will be outdated information as soon as a new line is added or removed in the file.
> 
> Something like this PR's title would be a clearer commit message: "Explicit error message when setting active a Viewport that is already active"

Sorry for mistakes. It should be right now.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.

**Comment by akien-mga** [comment]
Cherry-picked for 3.3.3.

**Pull request merged**